Makefile

```diff
@@ -48,6 +48,8 @@ audit-%:
 	npm --prefix $(subst sdk,open-api/typescript-sdk,$*) audit fix
 install-%:
 	npm --prefix $(subst sdk,open-api/typescript-sdk,$*) i
+cinstall-%:
```

Comment:
I think calling this `ci-%` would be clearer.

Reply:
Agreed